src/Cache.cpp

```diff
@@ -115,19 +115,19 @@ tr1::shared_ptr<Frame> Cache::GetSmallestFrame()
 	const GenericScopedLock<CriticalSection> lock(*cacheCriticalSection);
 	tr1::shared_ptr<openshot::Frame> f;
 
-	 // Loop through frame numbers
-	 deque<long int>::iterator itr;
-	 long int smallest_frame = -1;
-	 for(itr = frame_numbers.begin(); itr != frame_numbers.end(); ++itr)
-	 {
-		 if (*itr < smallest_frame || smallest_frame == -1)
-			 smallest_frame = *itr;
-	 }
+	// Loop through frame numbers
+	deque<long int>::iterator itr = frame_numbers.begin();
+	long int smallest_frame = *itr;
```

Comment:
Now I'm confused. I was pretty sure I had copied that model from somewhere else in Cache.cpp, but now I can't find it. :-(